build.gradle

```diff
@@ -89,7 +89,7 @@ subprojects {
 
                 // TODO: Unreleased dependencies.
                 // These must already be installed in the local maven repository.
-                netty: 'io.netty:netty-codec-http2:4.1.0.Beta5-SNAPSHOT',
+                netty: 'io.netty:netty-codec-http2:4.1.0.Beta6-SNAPSHOT',
```

Comment:
Yes they can do that ... but what's the longer-term plan for absorbing Netty changes?  Eventually we're going to need something from an unreleased version of Netty.  Or are you thinking that we just have to wait until the next beta release to absorb the changes (which can be weeks)?